src/blocks/content-helper/current-post-details/component.tsx

```diff
@@ -0,0 +1,341 @@
+							title={
+								/* translators: 1: Referrer type, 2: Percentage value, %%: Escaped percent sign */
+								sprintf( __( '%s: %s%%', 'wp-parsely' ), getKeyTitle( key ), value.viewsPercentage ) // eslint-disable-line @wordpress/valid-sprintf
+							}
+							style={ { width: value.viewsPercentage + '%' } }>
+						</div>
+					);
+				} ) }
+			</div>
+
+			<table>
+				<thead>
+					<tr>{
+						Object.keys( data.referrers.types ).map( ( key ) => {
+							return <th key={ key }>{ getKeyTitle( key ) }</th>;
+						} ) }
+					</tr>
+				</thead>
+				<tbody>
+					<tr>{
+						Object.entries( data.referrers.types ).map( ( [ key, value ] ) => {
+							return <td key={ key }>{ ImpreciseNumber( value.views ) }</td>;
+						} ) }
+					</tr>
+				</tbody>
+			</table>
+		</div>
+	);
+}
+
+/**
+ * Outputs the "Top Referrers" section.
+ *
+ * @param {PostPerformanceData} data The data needed to populate the section.
+ */
+function TopReferrersSection( data: PostPerformanceData ) {
+	let totalViewsPercentage = 0;
+	return (
+		<div className="section top-referrers">
+			<table>
+				<thead>
+					<tr>
+						<th>{ __( 'Top Referrers', 'wp-parsely' ) }</th>
+						<th colSpan={ 2 }>{ __( 'Page Views', 'wp-parsely' ) }</th>
+					</tr>
+				</thead>
+				<tbody>{
+					Object.entries( data.referrers.top ).map( ( [ key, value ] ) => {
+						if ( key === 'totals' ) {
+							totalViewsPercentage = Number( value.viewsPercentage );
+							return null;
+						}
+
+						let referrerUrl = key;
+						if ( key === 'direct' ) {
+							referrerUrl = __( 'Direct', 'wp-parsely' );
+						}
+
+						return (
+							<tr key={ key }>
+								<th title={ referrerUrl }>{ referrerUrl }</th>
+								<td title={
+									/* translators: %s: Percentage value, %%: Escaped percent sign */
+									sprintf( __( '%s%%', 'wp-parsely' ), value.viewsPercentage ) // eslint-disable-line @wordpress/valid-sprintf
+								}>
+									<div className="percentage-bar" style={ { '--bar-fill': value.viewsPercentage + '%' } as React.CSSProperties }></div>
+								</td>
+								<td>{ ImpreciseNumber( value.views ) }</td>
+							</tr>
+						);
+					} )
+				}
+				</tbody>
+			</table>
+			<div> {
+				/* translators: %s: Percentage value, %%: Escaped percent sign */
+				sprintf( __( '%s%% of views came from top referrers.', 'wp-parsely' ), totalViewsPercentage ) // eslint-disable-line @wordpress/valid-sprintf
+			}
+			</div>
+		</div>
+	);
+}
+
+/**
+ * Outputs the "Actions" section.
+ *
+ * @param {PostPerformanceData} data The data needed to populate the section.
+ */
+function ActionsSection( data: PostPerformanceData ) {
+	return (
+		<div className="section actions">
+			<Button href={ data.url } rel="noopener" target="_blank" variant="secondary">{ __( 'Visit Post', 'wp-parsely' ) }</Button>
+			<Button href={ data.statsUrl } rel="noopener" target="_blank" variant="primary">{ __( 'View in Parse.ly', 'wp-parsely' ) }</Button>
```

Comment:
We need there to be a `<span className="screen-reader-text"> { __( '(opens in new tab)', 'wp-parsely' ) }</span>` as part of the button text for both of these buttons. It's OK to keep these as a separately translated string in this case since they are self-contained.

Any link with a target of `_blank` should tell screen reader users that it will open a new tab so they understand why they've just been shifted away from the current tab.